zion/src/test/scala/edu/uci/ics/cloudberry/zion/model/impl/JSONParserTest.scala

```diff
@@ -34,5 +35,36 @@ class JSONParserTest extends Specification with TestQuery {
       val expectQuery = Query(schema.dataset, Seq.empty, filter, Seq.empty, None, Some(selectRecent))
       actualQuery must_== expectQuery
     }
+    "parse long values " in {
```

Comment:
nice, could we add all types?  `parse int `, `parse string`.
I haven't think about if `bag` is applicable for values, do you think it is reasonable to pass a array of `bag`s? if not we should throw warnings.